index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script src="index.js" charset="utf-8" defer></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Tic-tac-toe</title>
+</head>
+
+<body>
+  <div class="container">
+    <h1>Tic-tac-toe</h1>
+    <table>
```

Comment:
That's quite a good start .. but I would avoid tables.
Let's try to get to the same result without using tables and no position absolute. 😉